dragonfly/session/world.go

```diff
@@ -741,3 +759,8 @@ func vec32To64(vec3 mgl32.Vec3) mgl64.Vec3 {
 func vec64To32(vec3 mgl64.Vec3) mgl32.Vec3 {
 	return mgl32.Vec3{float32(vec3[0]), float32(vec3[1]), float32(vec3[2])}
 }
+
+// abs ...
+func abs(a int) int {
+	return int(math.Abs(float64(a)))
```

Comment:
Might be better not to use floats here. A simple `if a > 0 { return a } return -a` might be better.